### DISCUSSION TITLE
Videos don't show ads

### DISCUSSION THREAD

**Discussion by EHRdev** [LonelyCpp/react-native-youtube-iframe#228] [Q&A]
Hi !, more than an issue, it is just commenting that the videos that usually have ads (monetized) don't show any type of ads and I would like to know how this is possible? it's really amazing, congratulations

**Comment by LonelyCpp** [comment]
https://support.google.com/youtube/answer/132596?hl=en

> Ads appear on brand safe sites: YouTube works diligently so that our advertisers' brands appear on sites that reflect our respective core values. Our systems closely evaluate websites and their content against various factors when finding out whether to turn on In-stream ads on YouTube embeds. These factors include a strict set of guidelines on content like adult imagery, violence, inappropriate and hateful language, and sites that promote infringement.

**Comment by jake1271** [comment]
Anyone know if it's possible to have the ads play? I'd like to build an app but would want the creators to be rewarded too. I know it's possible because I've seen native Android apps do it.
But is it possible with RN?
